pkg/connection/config.go

```diff
@@ -31,13 +31,39 @@ type Config struct {
 	Token    string `mapstructure:"token"`    // bearer token
 }
 
+// OverrideActiveConfigRegistryProject forces the active configuration to use the specified project.
+func OverrideActiveConfigRegistryProject(project string) {
+	// force the config to be read and cached
+	useCache = true
+	_, err := ActiveConfig()
+	if err == nil && cachedConfig != nil {
+		cachedConfig.Project = project
+	}
+}
+
+var cachedConfig *Config
+var useCache bool
+
+func ClearCachedConfig() {
```

Comment:
What's the use case for this? I'm curious why it's added but not used.

Reply:
Oops, I'll take this out. I added it to try to fix other tests that were broken by this change but found that the `useCache` flag (hack) made this unnecessary.